containers/Dockerfile.buf

```diff
@@ -0,0 +1,21 @@
+# Setup Go alpine to install protoc-gen-go plugin.
+FROM golang:alpine AS golang
+ENV GO111MODULE=on
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.35.1
```

Comment:
The buf image doesn't come with ways to configure plugins so this is what was recommended by their team on various Github issues.